src/screens/conclusion/index.js

```diff
@@ -1,73 +1,85 @@
-import "./conclusion.css";
+import { PatreonIcon } from "../../components/patreon-icon";
 
 export function Conclusion() {
   return (
-    <section id="conclusion" className="together-we-can screen-content">
-      <div className="full-screen bears h-full w-full flex-col justify-between">
-        <div className="h-screen-50vh flex w-[100%] flex-col self-start py-16 pr-10 lg:w-[75%] 2xl:w-[50%]">
-          <h2>Together, we can make life better for wild animals.</h2>
+    <section
+      id="conclusion"
+      className="relative min-h-screen bg-woods bg-cover md:bg-woods-md md:bg-center"
+    >
+      <img
+        className="absolute right-20 top-52 w-96 lg:w-[500px]"
```

Comment:
Instead of using the arbitrary value of 500px, we could do something like this instead:
```tsx
className="… max-w-lg lg:w-full"
```

Reply:
We may also want to hide these images from screen readers, as it doesn't add anything. You can use `not-sr-only` for this.